examples/rest-json/src/main/java/org/acme/rest/json/Fruit.java

```diff
@@ -58,7 +58,7 @@ public boolean equals(Object obj) {
             return false;
         }
 
-        Fruit other = (Fruit) obj;
+        Fruit other = (Fruit)obj;
```

Comment:
Having a space after the cast seems to be more common, even in the main Camel code base.

The following setting would do that:

```
<setting id="org.eclipse.jdt.core.formatter.insert_space_after_closing_paren_in_cast" value="insert"/>
```